runtime/lua/vim/snippet.lua

```diff
@@ -446,14 +446,18 @@ function M.expand(input)
       base_indent = base_indent .. (snippet_lines[#snippet_lines]:match('(^%s*)%S') or '') --- @type string
     end
 
+    local sw = vim.fn.shiftwidth()
```

Comment:
Why use `vim.fn.shiftwidth` here and not `vim.bo[curbuf].shiftwidth` (as we do for `expandtab`)?

Reply:
`vim.fn.shiftwidth` also handles if `expandtab` is not enabled

Reply:
so it will use `tabstop` or `shiftwidth` depending on the input